BeagleBone_black/motor_control/main.cpp

```diff
@@ -0,0 +1,32 @@
+/*
+ * Author: Sean Mullan and Jack Horsburgh
+ * Organisation: HYPED
+ * Date: 17/02/18
+ * Description:
+ *
+ *    Copyright 2018 HYPED
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#include "motor.cpp"
+#include "motor_controller.cpp"
+#include <iostream>
+
+int main() {
```

Comment:
See https://hyp-ed.github.io/styleguide/#Function_Declarations_and_Definitions. (The opening brace is in a wrong place.)